### DISCUSSION TITLE
[BUG]: review comment cancel messages is null

### DISCUSSION THREAD

**Discussion by IbukiMoriya** [community/community#34507] [Pull Requests]
if click cancel in the first image, show null message

is it only my environment?

![image](https://user-images.githubusercontent.com/60495021/192913571-0f11c893-ab00-4a7a-8af7-aa7ba10d04ca.png)
![image](https://user-images.githubusercontent.com/60495021/192913466-d0049ca9-aed5-4976-9c7e-f3f82c1a2e8c.png)

**Comment by pythonpioneer** [comment]
You don't need to cancel it, just remove the text.
